fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableClauseAnalyzer.java

```diff
@@ -0,0 +1,37 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.google.common.base.Strings;
+import com.starrocks.analysis.AlterClause;
+import com.starrocks.analysis.TableRenameClause;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.FeNameFormat;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.AstVisitor;
+
+public class AlterTableClauseAnalyzer {
```

Comment:
Combine AlterStmtAnalyzerVisitor and AlterTableClauseAnalyzer into an AlterTableStatementAnalyzer.